docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -32,52 +32,70 @@ The following diagram provides an overview of the key GX Cloud architecture comp
+- **GX Cloud web UI** - A web interface that allows you to manage and validate your organization's data quality without running Python code and enables shared visibility into your organization's Validation Results and Checkpoint run history. It's browser- and platform-independent.
 
-- **GX Cloud API** - Provides a REST API to programmatically access and manage GX Cloud data and configurations. Both the GX open source software (OSS) client and the GX Agent use the GX Cloud API to query data from and send data to GX Cloud. Documentation for the GX Cloud API is not currently available, and it is not yet intended for use outside the GX OSS client.
+- **GX Cloud API** - Provides a REST API to programmatically access and manage GX Cloud data and configurations. Both the GX open source software (OSS) client and the GX Agent use the GX Cloud API to query data from and send data to GX Cloud. Documentation for the GX Cloud API is not currently available, and it is not yet intended for use outside the GX OSS client and the GX Agent.
 
-- **GX Cloud message broker** - Enables communication between GX Cloud and your local GX Agent.
+- **GX Cloud message broker** - Enables communication between GX Cloud and the GX Agent.
 
-### Local components
+### Local and deployment components
 
-- **GX OSS client** - The Python library that powers GX Cloud and provides a Python client for GX Cloud when you're using GX OSS locally. GX OSS contains the logic needed to test and document your data, and you can also use it to create, manage, and interact with GX Cloud components.
+- **GX OSS client** - The Python library that powers GX Cloud and provides a Python client for programmatic access to GX Cloud. GX OSS contains the logic needed to test and document your organization's data, and you can also use it to create, manage, and interact with GX Cloud components.
 
-- **GX Agent** - A utility that runs locally in your environment. When it's running, the GX Agent can receive tasks generated from the GX Cloud web UI, such as running a Checkpoint or fetching Column Descriptive Metrics, and execute these tasks against your Data Assets.
+- **GX Agent** - A utility that runs in your organization's deployment environment. While running, the GX Agent can receive tasks generated from the GX Cloud web UI, such as running a Checkpoint or fetching Metrics, and execute these tasks against your Data Assets.
+
+## GX Agent
+
+The GX Agent is an intermediary between GX Cloud and your organization's data stores. The GX Agent performs jobs initiated from the GX Cloud web UI; GX Cloud does not connect directly to your data. All data access occurs within the GX Agent, and the GX Agent sends job results to GX Cloud.
+
+The GX Agent is [typically deployed in your organization's cloud services environment](./connect/connect_snowflake.md#deploy-the-gx-agent) and serves all GX Cloud users within your organization. It can be run as part of development or production workflows.
```

Comment:
```suggestion
The GX Agent is [typically deployed in your organization's deployment environment](./connect/connect_lp) and serves all GX Cloud users within your organization. It can be run as part of your development or production workflows.
```